go.mod

```diff
@@ -1,5 +1,7 @@
 module github.com/fission/fission
 
+go 1.15
```

Comment:
We will have to switch to version that is in use right now - for now I think is 1.12.X - can you please change that?